lib/image.js

```diff
@@ -85,3 +79,11 @@ Image.prototype.inspect = function(){
     + (this.complete ? ' complete' : '')
     + ']';
 };
+
+Image.prototype.onHttpError = function(err) {
```

Comment:
I don't think there should be a new prototype method on Image. We want to stick as closely to the HTML Image definition as possible. Could you restore the `onerror` bit to how it was so that it remains hidden please?

Reply:
Done